provision/aws/rosa_create_cluster.sh

```diff
@@ -90,3 +90,4 @@ echo
 rosa create machinepool -c "${CLUSTER_NAME}" --instance-type m5.4xlarge --max-replicas 0 --min-replicas 0 --name scaling --use-spot-instances --enable-autoscaling
 
 ./rosa_efs_create.sh
+./rosa_install_ispn_operator.sh
```

Comment:
I wasn't planning to add cross-site to Minikube simply because I don't see any value in running a cross-site benchmark locally. Also, it requires some work and configuration [1].
`provision/openshift` is run every time a Keycloak cluster is deployed in OpenShift and we probably want to install the operator only once and not every run. Said that, I'm planning to create `provision/infinispan` to put the scripts and configuration to deploy Infinispan with cross-site; it may be worth moving it there.

[1] https://minikube.sigs.k8s.io/docs/handbook/accessing/

Reply:
I'd be ok if we get it working in OpenShift first, and then see if this makes sense in a minikube environment as well. 

Note that simulating cross DC within one OpenShift cluster might only be the first step. I assume we eventually want to have multiple OpenShift clusters in multiple regions that somehow connect? Let's discuss this later today on our team meeting.

Reply:
> > I wasn't planning to add cross-site to Minikube
> 
> Won't we need the Infinispan operator for the non-xsite use-case where Infinispan is external to keycloak?
> 
I'm not sure, to be honest. As far as I know, that is not a supported use case and we only externalize the sessions for cross-site support. @ahus1 Do you know anything about this?

Reply:
> Note that simulating cross DC within one OpenShift cluster might only be the first step. I assume we eventually want to have multiple OpenShift clusters in multiple regions that somehow connect? Let's discuss this later today on our team meeting.

Exactly, having multiple OpenShift clusters is the way to go. I'll add cross-site Infinispan in a single cluster (each Infinispan deployment is in a separate namespace and they are connected using cross-site) just because it is simpler enough to do it :+1:

Reply:
@pruivo / @ryanemerson - we might look into such a scenario when (if) there are scenarios/benchmarks that show that the internal Infinispan doesn't deliver the results we want to see. 

The only external Infinispan setup we currently plan for is the cross-site "by the book" deployment for now where we then do some benchmarks. Once we have scenarios where the load tests show failures, we start optimizing.

Reply:
Good to know, thanks for clarifying.